org.gnome.gThumb.yml

```diff
@@ -144,14 +122,11 @@ modules:
       - install -Dm644 ../data/appdata/org.gnome.gThumb.appdata.xml.in $FLATPAK_DEST/share/appdata/$FLATPAK_ID.appdata.xml
```

Comment:
```
Exporting share/appdata/org.gnome.gThumb.appdata.xml
Exporting share/metainfo/org.gnome.gThumb.appdata.xml
```

there is defintely one too many

Upstream does:
```
Installing data/appdata/org.gnome.gThumb.appdata.xml to /app/share/metainfo
```
so this should be removed here.